ironfish/src/storage/database/encoding.ts

```diff
@@ -50,6 +50,26 @@ export class StringHashEncoding implements IDatabaseEncoding<string> {
   }
 }
 
+export class NullableStringEncoding implements IDatabaseEncoding<string | null> {
```

Comment:
This is pretty much a copy-paste of the `MetaValueEncoding`. Whether we want to hard-code the meta db values to use this new encoding, or allowing it to have the freedom to be extended as needed can be a discussion for a later time, I guess, unless anyone feels strongly enough about it.